help/destinations/catalog/data-partners/merkury-enterprise-connections.md

```diff
@@ -0,0 +1,245 @@
+generated](./merkury-connections/media/image6.png){width="6.5in"
+height="3.877083333333333in"}
+
+-   **Name (Required)** - The name the destination will be saved under
+
+-   **Description** - Short explanation of the destination's purpose
+
+-   **Bucket Name (Required)** - Name of the Amazon S3 bucket set up on
+    S3
+
+-   **Folder Path (Required)** - If subdirectories in a bucket are used
+    a path must be defined, or '/' to reference the root path.
+
+-   **File Type** - Select the format Experience Platform should use for
+    the exported files. Currently, the only file type Acxiom processing
+    will be expecting is CSV
+
+![A white background with black text Description automatically
+generated](./merkury-connections/media/image7.png){width="6.5in"
+height="1.336111111111111in"}
+
+![A screenshot of a computer Description automatically
+generated](./merkury-connections/media/image8.png){width="4.875in"
+height="7.791666666666667in"}
+
+## **Enable alerts
+
+You can enable alerts to receive notifications on the status of the
+dataflow to your destination. Select an alert from the list to subscribe
+to receive notifications on the status of your dataflow. For more
+information on alerts, see the guide on [subscribing to destinations
+alerts using the
+UI](https://experienceleague.adobe.com/en/docs/experience-platform/destinations/ui/alerts).
+
+When you are finished providing details for your destination connection,
+select **Next**.
+
+## Activate audiences to this destination
+
+![A screenshot of a computer screen Description automatically
+generated](./merkury-connections/media/image9.png){width="7.04362532808399in"
+height="2.709839238845144in"}
+
+Read [Activate audience data to batch profile export
+destinations](https://experienceleague.adobe.com/en/docs/experience-platform/destinations/ui/activate/activate-batch-profile-destinations) for
+instructions on activating audiences to this destination.
+
+## Mapping suggestions
+
+The correct processing of files on the Merkury side requires name and
+address elements. While not all elements are required, providing as much
+as possible will aid in successful matching.
+
+Mapping suggestions are provided in the table below listing attributes
+on your destination side that are used by Merkury processing that
+customers can map profile attributes to. Treat these elements as
+suggestions as not all elements are required, and the source values will
+depend on the needs of the account.
+
+![A screenshot of a computer Description automatically
+generated](./merkury-connections/media/image10.png){width="6.875977690288714in"
+height="3.510715223097113in"}
+
+## Validate data export
+
+To verify if data has been exported successfully, check your Amazon S3
+Storage bucket, and make sure that the exported files contain the
+expected profile populations.
+
+## Next steps
+
+By following this tutorial, you have successfully created a dataflow to
```

Comment:
Again, I notice that this is copied from https://experienceleague.adobe.com/en/docs/experience-platform/destinations/catalog/data-partner/acxiom-data-enhancement#exported-data . Is the process the same for  the Merkury destination?

Reply:
Addressed